tests/perf/benchmark.py

```diff
@@ -128,23 +136,26 @@ def __init__(
         self.deterministic = deterministic
         self.accelerator = accelerator
         self.reference_results = reference_results
+        self.resume_from = resume_from
+        if (test_only == "export" and eval_upto == "train") or (
```

Comment:
How about using enum instead of comparing strings?

Reply:
It's a fine idea but there is some ambiguous point there. `eval_upto` is expected to get `train`, `export` or `optimize` but `resume_from` can also get `all` including them. So, even if there is enum class, using it for both of them looks a little bit weird. Also comparing between values including `all` is also other ambiguous point. For these reasons, I think it's better to leave it as it is for now.